<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindows</class>
 <widget class="QMainWindow" name="MainWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>S.U.S.A.N. Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>166</width>
      <height>25</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="box_instrument">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>141</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Instrument</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="piano_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Piano</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="guitar_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Guitar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="drum_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Drum</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="violin_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Violin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trumpet_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Trumpet</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="electric_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Electric_Bass</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_pitch">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>131</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Pitch</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="perfect_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Perfect</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="absolute_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Absolute</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="sharp_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sharp</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="flat_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Flat</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="diatonic_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Diatonic</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_rhythm">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>111</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Rhythm</string>
    </property>
    <widget class="QLabel" name="max_bpm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>208 bpm</string>
     </property>
    </widget>
    <widget class="QSlider" name="rhythm_slicer">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>59</y>
       <width>20</width>
       <height>171</height>
      </rect>
     </property>
     <property name="minimum">
      <number>40</number>
     </property>
     <property name="maximum">
      <number>208</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="min_bpm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>40 bpm</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_volume">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>111</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Volume</string>
    </property>
    <widget class="QLabel" name="min_volume">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
    <widget class="QLabel" name="max_volume">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100 dB</string>
     </property>
    </widget>
    <widget class="QSlider" name="volume_slicer">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>20</width>
       <height>171</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to SUSAN Music Interface</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select your music input to start simulation!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_speak">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Do you want to hear the robot speak out loud?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_emotion">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Do you want the robot show its emotion?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_robot">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Do you have a Pepper robot interface pluged?</string>
    </property>
   </widget>
   <zorder>box_instrument</zorder>
   <zorder>buttonBox</zorder>
   <zorder>box_pitch</zorder>
   <zorder>box_rhythm</zorder>
   <zorder>box_volume</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>check_speak</zorder>
   <zorder>check_emotion</zorder>
   <zorder>check_robot</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
